### PR TITLE
replaced references to Cortex with Mimir

### PR DESCRIPTION
**Pull request by achatterjee-grafana**
This PR replaces references to Cortex with Mimir in Alerting documentation. 

The only places I have left them are in release notes for 8.1 and 8.2.

### CODE REVIEW

**Comment by achatterjee-grafana** [comment]
Question for @09jvilla - should we backport the docs change to 8.4?

**Comment by 09jvilla** [comment]
Please refer to it as `Grafana Mimir` please. 

I'll let @armandgrillet chime in on the version - it depends on whichever version https://github.com/grafana/alerting-squad/issues/219 gets released into

**Comment by osg-grafana** [comment]
> Question for @09jvilla & @armandgrillet - should we backport the docs change to 8.4? Also, should we refer to the product as Mimir or Grafana Mimir in alerting docs?

Grafana Mimir in alerting docs.

File: docs/sources/alerting/_index.md
Comment (osg-grafana) [inline docs/sources/alerting/_index.md:13]:
```suggestion
- Create and manage Grafana Mimir and Loki managed alerts
```

File: docs/sources/alerting/unified-alerting/_index.md
Comment (osg-grafana) [inline docs/sources/alerting/unified-alerting/_index.md:14]:
```suggestion
- [Create Grafana Mimir or Loki managed alerting rules]({{< relref "alerting-rules/create-mimir-loki-managed-rule.md" >}})
```

File: docs/sources/alerting/unified-alerting/alerting-rules/_index.md
Comment (osg-grafana) [inline docs/sources/alerting/unified-alerting/alerting-rules/_index.md:15]:
```suggestion
- [Create Grafana Mimir or Loki managed alert rule]({{< relref "./create-mimir-loki-managed-rule.md" >}})
```

File: docs/sources/alerting/unified-alerting/alerting-rules/_index.md
Comment (osg-grafana) [inline docs/sources/alerting/unified-alerting/alerting-rules/_index.md:17]:
```suggestion
- [Create Grafana Mimir or Loki managed recording rule]({{< relref "./create-mimir-loki-managed-recording-rule.md" >}})
- [Edit Grafana Mimir or Loki rule groups and namespaces]({{< relref "./edit-mimir-loki-namespace-group.md" >}})
```

File: docs/sources/alerting/unified-alerting/alerting-rules/create-cortex-loki-managed-recording-rule.md
Comment (osg-grafana) [inline docs/sources/alerting/unified-alerting/alerting-rules/create-cortex-loki-managed-recording-rule.md:3]:
```suggestion
title = "Create Grafana Mimir or Loki managed recording rule"
description = "Create Grafana Mimir or Loki managed recording rule"
```

File: docs/sources/alerting/unified-alerting/alerting-rules/create-cortex-loki-managed-recording-rule.md
Comment (osg-grafana) [inline docs/sources/alerting/unified-alerting/alerting-rules/create-cortex-loki-managed-recording-rule.md:14]:
```suggestion
For Grafana Mimir and Loki data sources to work with Grafana 8.0 alerting, enable the ruler API by configuring their respective services.
```

File: docs/sources/alerting/unified-alerting/alerting-rules/create-cortex-loki-managed-recording-rule.md
Comment (osg-grafana) [inline docs/sources/alerting/unified-alerting/alerting-rules/create-cortex-loki-managed-recording-rule.md:18]:
```suggestion
**Grafana Mimir** - When configuring a Grafana Prometheus data source to point to Grafana Mimir, use the legacy `/api/prom` prefix, not `/prometheus`. Currently, we support only single-binary mode and you cannot provide a separate URL for the ruler API.
```

File: docs/sources/alerting/unified-alerting/alerting-rules/create-cortex-loki-managed-recording-rule.md
Comment (osg-grafana) [inline docs/sources/alerting/unified-alerting/alerting-rules/create-cortex-loki-managed-recording-rule.md:22]:
```suggestion
## Add a Grafana Mimir or Loki managed recording rule
```

File: docs/sources/alerting/unified-alerting/alerting-rules/create-cortex-loki-managed-recording-rule.md
Comment (osg-grafana) [inline docs/sources/alerting/unified-alerting/alerting-rules/create-cortex-loki-managed-recording-rule.md:28]: I think it is okay to leave this as Mimir instead of Grafana Mimir.

File: docs/sources/alerting/unified-alerting/alerting-rules/create-cortex-loki-managed-rule.md
Comment (osg-grafana) [inline docs/sources/alerting/unified-alerting/alerting-rules/create-cortex-loki-managed-rule.md:3]:
```suggestion
title = "Create Grafana Mimir or Loki managed alert rule"
description = "Create Grafana Mimir or Loki managed alerting rule"
```

File: docs/sources/alerting/unified-alerting/alerting-rules/create-cortex-loki-managed-rule.md
Comment (osg-grafana) [inline docs/sources/alerting/unified-alerting/alerting-rules/create-cortex-loki-managed-rule.md:8]:
```suggestion
# Create a Grafana Mimir or Loki managed alerting rule
```

File: docs/sources/alerting/unified-alerting/alerting-rules/create-cortex-loki-managed-rule.md
Comment (osg-grafana) [inline docs/sources/alerting/unified-alerting/alerting-rules/create-cortex-loki-managed-rule.md:10]:
```suggestion
Grafana allows you to create alerting rules for an external Grafana Mimir or Loki instance.
```

Comment (osg-grafana) [reply docs/sources/alerting/unified-alerting/alerting-rules/create-cortex-loki-managed-rule.md:10]: @09jvilla is "instance" accurate here for Mimir?

File: docs/sources/alerting/unified-alerting/alerting-rules/create-cortex-loki-managed-rule.md
Comment (osg-grafana) [inline docs/sources/alerting/unified-alerting/alerting-rules/create-cortex-loki-managed-rule.md:14]:
```suggestion
- Verify that you have write permission to the Prometheus data source. Otherwise, you will not be able to create or update Grafana Mimir managed alerting rules.
```

File: docs/sources/alerting/unified-alerting/alerting-rules/create-cortex-loki-managed-rule.md
Comment (osg-grafana) [inline docs/sources/alerting/unified-alerting/alerting-rules/create-cortex-loki-managed-rule.md:16]:
```suggestion
- For Grafana Mimir and Loki data sources, enable the ruler API by configuring their respective services.
```

File: docs/sources/alerting/unified-alerting/alerting-rules/create-cortex-loki-managed-rule.md
Comment (osg-grafana) [inline docs/sources/alerting/unified-alerting/alerting-rules/create-cortex-loki-managed-rule.md:20]:
```suggestion
  - **Grafana Mimir** - use the [legacy `/api/prom` prefix](https://grafana.com/docs/mimir/latest/operators-guide/reference-http-api/#path-prefixes), not `/prometheus`. The Prometheus data source supports both Grafana Mimir and Prometheus, and Grafana expects that both the [Query API](https://grafana.com/docs/mimir/latest/operators-guide/reference-http-api/#querier--query-frontend) and [Ruler API](https://grafana.com/docs/mimir/latest/operators-guide/reference-http-api/#ruler) are under the same URL. You cannot provide a separate URL for the Ruler API.
```

File: docs/sources/alerting/unified-alerting/alerting-rules/create-cortex-loki-managed-rule.md
Comment (osg-grafana) [inline docs/sources/alerting/unified-alerting/alerting-rules/create-cortex-loki-managed-rule.md:24]:
```suggestion
## Add a Grafana Mimir or Loki managed alerting rule
```

File: docs/sources/alerting/unified-alerting/alerting-rules/create-cortex-loki-managed-rule.md
Comment (osg-grafana) [inline docs/sources/alerting/unified-alerting/alerting-rules/create-cortex-loki-managed-rule.md:30]: Likely fine as Mimir.

File: docs/sources/alerting/unified-alerting/alerting-rules/create-cortex-loki-managed-rule.md
Comment (osg-grafana) [inline docs/sources/alerting/unified-alerting/alerting-rules/create-cortex-loki-managed-rule.md:32]:
```suggestion
   - From the **Namespace** drop-down, select an existing rule namespace. Otherwise, click **Add new** and enter a name to create a new one. Namespaces can contain one or more rule groups and only have an organizational purpose. For more information, see [Grafana Mimir or Loki rule groups and namespaces]({{< relref "./edit-mimir-loki-namespace-group.md" >}}).
```

File: docs/sources/alerting/unified-alerting/alerting-rules/edit-cortex-loki-namespace-group.md
Comment (osg-grafana) [inline docs/sources/alerting/unified-alerting/alerting-rules/edit-cortex-loki-namespace-group.md:2]:
```suggestion
title = "Grafana Mimir or Loki rule groups and namespaces"
```

File: docs/sources/alerting/unified-alerting/alerting-rules/edit-cortex-loki-namespace-group.md
Comment (osg-grafana) [inline docs/sources/alerting/unified-alerting/alerting-rules/edit-cortex-loki-namespace-group.md:3]:
```suggestion
description = "Edit Grafana Mimir or Loki rule groups and namespaces"
```

File: docs/sources/alerting/unified-alerting/alerting-rules/edit-cortex-loki-namespace-group.md
Comment (osg-grafana) [inline docs/sources/alerting/unified-alerting/alerting-rules/edit-cortex-loki-namespace-group.md:4]:
```suggestion
keywords = ["grafana", "alerting", "guide", "group", "namespace", "grafana mimir", "loki"]
```

File: docs/sources/alerting/unified-alerting/alerting-rules/edit-cortex-loki-namespace-group.md
Comment (osg-grafana) [inline docs/sources/alerting/unified-alerting/alerting-rules/edit-cortex-loki-namespace-group.md:8]:
```suggestion
# Grafana Mimir or Loki rule groups and namespaces
```

File: docs/sources/alerting/unified-alerting/alerting-rules/edit-cortex-loki-namespace-group.md
Comment (osg-grafana) [inline docs/sources/alerting/unified-alerting/alerting-rules/edit-cortex-loki-namespace-group.md:10]:
```suggestion
A namespace contains one or more groups. The rules within a group are run sequentially at a regular interval. The default interval is one (1) minute. You can rename Grafana Mimir or Loki rule namespaces and groups, and edit group evaluation intervals.
```

File: docs/sources/alerting/unified-alerting/alerting-rules/edit-cortex-loki-namespace-group.md
Comment (osg-grafana) [inline docs/sources/alerting/unified-alerting/alerting-rules/edit-cortex-loki-namespace-group.md:21]:
```suggestion
1. Find a Grafana Mimir or Loki managed rule with the group that belongs to the namespace you want to edit.
```

File: docs/sources/alerting/unified-alerting/alerting-rules/edit-cortex-loki-namespace-group.md
Comment (osg-grafana) [inline docs/sources/alerting/unified-alerting/alerting-rules/edit-cortex-loki-namespace-group.md:32]:
```suggestion
1. Find a Grafana Mimir or Loki managed rule with the group you want to edit.
```

File: docs/sources/alerting/unified-alerting/alerting-rules/rule-list.md
Comment (osg-grafana) [inline docs/sources/alerting/unified-alerting/alerting-rules/rule-list.md:12]:
```suggestion
The **Mimir/Loki** rules section lists all rules for external Prometheus or Loki data sources. Cloud alerting rules are also listed in this section.
```

File: docs/sources/alerting/unified-alerting/alerting-rules/rule-list.md
Comment (osg-grafana) [inline docs/sources/alerting/unified-alerting/alerting-rules/rule-list.md:53]:
```suggestion
Grafana managed alerting rules can only be edited or deleted by users with Edit permissions for the folder storing the rules. Alerting rules for an external Grafana Mimir or Loki instance can be edited or deleted by users with Editor or Admin roles.
```

File: docs/sources/alerting/unified-alerting/alerting-rules/rule-list.md
Comment (osg-grafana) [inline docs/sources/alerting/unified-alerting/alerting-rules/rule-list.md:57]:
```suggestion
1. Click **Edit** to open the create rule page. Make updates following instructions in [Create a Grafana managed alerting rule]({{< relref "./create-grafana-managed-rule.md" >}}) or [Create a Grafana Mimir or Loki managed alerting rule]({{< relref "./create-mimir-loki-managed-rule.md" >}}).
```

File: docs/sources/alerting/unified-alerting/difference-old-new.md
Comment (osg-grafana) [inline docs/sources/alerting/unified-alerting/difference-old-new.md:22]:
```suggestion
## Create Loki and Grafana Mimir alerting rules
```

File: docs/sources/alerting/unified-alerting/difference-old-new.md
Comment (osg-grafana) [inline docs/sources/alerting/unified-alerting/difference-old-new.md:24]:
```suggestion
In Grafana alerting, you can manage Loki and Grafana Mimir alerting rules using the same UI and API as your Grafana managed alerts.
```

File: docs/sources/alerting/unified-alerting/fundamentals/alertmanager.md
Comment (osg-grafana) [inline docs/sources/alerting/unified-alerting/fundamentals/alertmanager.md:15]:
```suggestion
Grafana alerting added support for external Alertmanager configuration. When you add an [Alertmanager data source]({{< relref "../../../datasources/alertmanager.md" >}}), the Alertmanager drop-down shows a list of available external Alertmanager data sources. Select a data source to create and manage alerting for standalone Grafana Mimir or Loki data sources.
```

File: docs/sources/datasources/alertmanager.md
Comment (osg-grafana) [inline docs/sources/datasources/alertmanager.md:15]:
```suggestion
[Prometheus](https://prometheus.io/) and [Grafana Mimir](https://grafana.com/docs/mimir/latest/) (default) implementations of Alertmanager are supported. You can specify implementation in the data source settings page. In case of Prometheus contact points and notification policies are read-only in the Grafana alerting UI, as it does not support updating configuration via HTTP API.
```

File: docs/sources/datasources/alertmanager.md
Comment (osg-grafana) [inline docs/sources/datasources/alertmanager.md:32]: Is this correct?

File: docs/sources/datasources/prometheus.md
Comment (osg-grafana) [inline docs/sources/datasources/prometheus.md:203]:
```suggestion
- [Grafana Mimir](https://grafana.com/docs/mimir/latest/)
```

**Review by osg-grafana — APPROVED** [review]
Added Grafana in front of the instances of Mimir that need it. The menu items likely do not need to change. Approving to unblock.

File: docs/sources/alerting/unified-alerting/alerting-rules/_index.md
Comment (chri2547) [inline docs/sources/alerting/unified-alerting/alerting-rules/_index.md:16]: Should all bullet points be "Grafana Mimir"?

File: docs/sources/alerting/unified-alerting/alerting-rules/create-cortex-loki-managed-rule.md
Comment (chri2547) [reply docs/sources/alerting/unified-alerting/alerting-rules/create-cortex-loki-managed-rule.md:30]: Agree, the content should match the UI.

File: docs/sources/alerting/unified-alerting/alerting-rules/create-cortex-loki-managed-recording-rule.md
Comment (chri2547) [inline docs/sources/alerting/unified-alerting/alerting-rules/create-cortex-loki-managed-recording-rule.md:8]:
```suggestion
# Create a Grafana Mimir or Loki managed recording rule
```

File: docs/sources/alerting/unified-alerting/alerting-rules/create-cortex-loki-managed-recording-rule.md
Comment (chri2547) [inline docs/sources/alerting/unified-alerting/alerting-rules/create-cortex-loki-managed-recording-rule.md:10]:
```suggestion
You can create and manage recording rules for an external Grafana Mimir or Loki instance. Recording rules calculate frequently needed expressions or computationally expensive expressions in advance and save the result as a new set of time series. Querying this new time series is faster, especially for dashboards since they query the same expression every time the dashboards refresh.
```

**Review by chri2547 — APPROVED** [review]

**Comment by achatterjee-grafana** [comment]
> This is 8.5 only as #47063 was not backported to 8.4.

Thank you for the clarification!

File: docs/sources/datasources/alertmanager.md
Comment (gotjosh) [reply docs/sources/datasources/alertmanager.md:32]: I believe this is not correct but @JohnnyQQQQ or @alexweav can correct me if I'm wrong.

File: docs/sources/alerting/unified-alerting/alerting-rules/_index.md
Comment (JohnnyQQQQ) [inline docs/sources/alerting/unified-alerting/alerting-rules/_index.md:15]: I think this will break the link as we only rename the file reference and not the underlying file.

Comment (achatterjee-grafana) [reply docs/sources/alerting/unified-alerting/alerting-rules/_index.md:15]: On it.

**Pull request merged**
